package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "music-api",
+  "version": "1.0.0",
+  "description": "Build front end for Music-Api site. This music Api provides music albums for different genres.",
+  "main": "server.js",
+  "scripts": {
+    "dev": "./node_modules/.bin/concurrently \"npm run dev:client\" \"npm run dev:server\"",
+    "dev:client": "./node_modules/.bin/webpack --progress --inline --config webpack.dev.js",
+    "dev:server": "./node_modules/.bin/env-cmd ./node_modules/.bin/babel-watch server.js",
+    "dev:migrate": "./node_modules/.bin  /env-cmd ./node_modules/.bin/sequelize db:migrate --env development --config ./configs/sequelize.js",
```

Comment:
This typo caused your migrations to fail. You should be testing these things before submitting.